### DISCUSSION TITLE
Deploying to AWS without Keycloak and use AWS Cognito for authetication

### DISCUSSION THREAD

**Discussion by narramadan** [ApicurioArchive/apicurio-studio#2128] [Q&A]
Hi,

I am exploring options on removing dependency with Keycloak and add capability to use AWS Cognito for authentication.

Landed to this previous question - https://github.com/Apicurio/apicurio-studio/issues/964.

If you could guide what and where it would require change to remove keycloak dependency, I would like to try and integrate with AWS Cognito or other services.

Thanks,
Madan N

**Comment by EricWittmann** [comment]
I realize I'm a year late to answer this, but Studio 0.2 is quite dependent on Keycloak.  We're just now rebooting this project (Studio 1.0) and we should be able to address this kind of thing (auth support via essentially any OIDC provider).  It will come with a price though, which is that the GitHub integration will be missing at least to start.
